workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-cron/build.gradle

```diff
@@ -17,6 +17,12 @@ apply plugin: "com.liferay.source.formatter"
 apply plugin: "java"
 apply plugin: "org.springframework.boot"
 
+bootRun {
+	String virtualInstanceId = project.hasProperty("virtualInstanceId") ? project.getProperty("virtualInstanceId") : "default"
+	environment "LIFERAY_ROUTES_CLIENT_EXTENSION", "${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/${project.name}/"
+	environment "LIFERAY_ROUTES_DXP", "${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/dxp/"
+}
+
```

Comment:
This will eventually be integrated into the gradle workspace plugin right? find all the exec tasks and inject the env into them?

Reply:
correct that will happen in subsequent pull.